dist/generator-vulcanjs/lib/VulcanGenerator.js

```diff
@@ -85,8 +85,10 @@ module.exports = class VulcanGenerator extends Generator {
     errorsArr.forEach((error, index) => {
       const errorNo = `Error (${index})`;
       const message = `\n${errorNo}: ${chalk.red(error.message)}`;
-      this.env.error(message);
+      this.log(message);
```

Comment:
you have "non catched exceptions" otherwise. Try out, you now have working error messages without a full stack, which looks messy.